docs/source/user-guide/expressions.md

```diff
@@ -213,14 +233,14 @@ Unlike to some databases the math functions in Datafusion works the same way as
 
 ## Regular Expressions
 
-| Function       | Notes                                                                         |
+| Function       | Explanation                                                                   |
```

Comment:
Prob Description will be a better word here? 🤔

Reply:
The column changed form `Notes` to `Explanation` because some note admonitions added to this page. `Description` would also work for me.